docs/user/security/securing-communications/index.asciidoc

```diff
@@ -26,65 +26,65 @@ NOTE: You do not need to enable the {es} {security-features} for this type of en
 When you obtain a server certificate, you must set its subject alternative name (SAN) correctly to ensure that modern web browsers with
 hostname verification will trust it. You can set one or more SANs to the {kib} server's fully-qualified domain name (FQDN), hostname, or IP
 address. When choosing the SAN, you should pick whichever attribute you will be using to connect to {kib} in your browser, which is likely
-the FQDN.
+the FQDN in a production environment.
 
-
-You may choose to generate a certificate signing request (CSR) and private key using the {ref}/certutil.html[`elasticsearch-certutil`] tool.
+You may choose to generate a signed certificate and private key using the [`elasticsearch-certutil`] tool.
```

Comment:
As suggested by Tim in the linked comment, I rewrote these instructions to simply generate a certificate for local usage, with an added note that it is not appropriate for a production environment. I think this suffices, but it's really not any less wordy. 

To make this a bit more concise we could also consider:
* Move said note to the bottom of the section
* In Step 2, remove the subsection for PEM certificates (since our example now explicitly generates a P12 file)

Thoughts?

Reply:
I'm personally OK with the length, and I like the simpler steps to generate a certificate for local usage, too